tasks/main-blivet.yml

```diff
@@ -52,7 +52,7 @@
 - name: Apply defaults to pools and volumes [4/6]
   set_fact:
     _storage_vols_no_defaults_by_pool: "{{ _storage_vols_no_defaults_by_pool|default({})|
-                                        combine({item.name: _storage_vols_w_defaults|selectattr('pool', 'equalto', item.name)|list}) }}"
+                                        combine({item.name: _storage_vols_w_defaults|json_query('[?pool==`\"{}\"`]'.format(item.name))}) }}"
```

Comment:
I suggest to leave the original selectattr filter in a comment above, since it is probably more readable.